addons/purchase/report/purchase_report.py

```diff
@@ -121,6 +133,31 @@ def _where(self) -> SQL:
             """,
         )
 
+    def _join_days_to_arrival(self) -> SQL:
+        return SQL(
+            """
+            LEFT JOIN (
+                SELECT MIN(picking.date_done)                       AS date_done,
+                       purchase.id                                  AS purchase_id
+                FROM purchase_order 							    AS purchase
+                JOIN purchase_order_line						    AS order_line
+                    ON order_line.order_id = purchase.id
+                JOIN stock_move									    AS move
+                    ON move.purchase_line_id = order_line.id
+                JOIN stock_picking								    AS picking
```

Comment:
`purchase` doesn't depend on `stock`
this should probably move to `purchase_stock`

Reply:
PR to fix it: https://github.com/odoo/odoo/pull/176195